src/Diff.re

```diff
@@ -28,25 +28,58 @@ module MakeDiff = (IO1: ImageIO.ImageIO, IO2: ImageIO.ImageIO) => {
       let row = IO1.readRow(base, y);
       let row2 = IO2.readRow(comp, y);
 
+      let getBaseColor = (x, rowToGet) => {
+        IO1.readImgColor(
+          x,
+          rowToGet == y ? row : IO1.readRow(base, rowToGet),
```

Comment:
I don't really like, that we are getting the previous & next row on every AA check. 
I tried to implement some caching with an array or hashtable, but that seems to just slow things down.

If you have an idea, I would be happy to implement it 🙂

Reply:
I don't think that we need it, cause it should pretty fast direct memory access. But I will test it, it may be some overhead with creating new ocaml bigarry